tools/ibc-test/README.md

```diff
@@ -36,13 +36,9 @@ cd $AXON_SRC_PATH
 git checkout forcerelay-test
 cargo install --path .
 
-# setup hermes envs
```

Comment:
If ckb is involved, why not add the commands to install ckb.
Only axon is mentioned.